packages/next/src/client/components/static-generation-bailout.ts

```diff
@@ -58,9 +58,6 @@ export const staticGenerationBailout: StaticGenerationBailout = (
     link: 'https://nextjs.org/docs/messages/dynamic-server-error',
   })
 
-  // If postpone is available, we should postpone the render.
-  staticGenerationStore.postpone?.(reason)
```

Comment:
PPR depends on this being available and firing, is this called elsewhere?

Reply:
I think this is now more explicitly tracked in the new dynamic helpers like `trackDynamicDataAccessed`